vowpalwabbit/slim/include/vw_slim_predict.h

```diff
@@ -416,7 +416,14 @@ class vw_predict
       ns_copy_guard->feature_push_back(1.f, (constant << _stride_shift) + ex.ft_offset);
     }
 
-    score = GD::inline_predict<W>(*_weights, false, _ignore_linear, _interactions, /* permutations */ false, ex);
+    // create temporary interactions of type vector<vector<uint8_t>> to feed in GD::inline_predict
```

Comment:
This is a hot function. It is not acceptable to create a copy of the interactions on each run, can you find a way to fix this?

Reply:
Should be resolved now. I changed interactions type in `vw_slim_predict.h` from `vector<string>` to `vector<vector<uint8_t>>`. This required overloading the`find_opt` function. Could you check if this would be ok?